src/provider/element-templates/ElementTemplates.js

```diff
@@ -194,8 +209,87 @@ export default class ElementTemplates {
 
     this._commandStack.execute('propertiesPanel.camunda.changeTemplate', context);
 
+    this._eventBus.fire(`elementTemplates.${action}`, payload);
+
     return context.element;
```

Comment:
This is where we already return the "changed element" :)

Cf. https://github.com/bpmn-io/bpmn-js-properties-panel/pull/935/files#r1224061460.